project_security_group/security/project_security_group_security.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+    <!-- GROUPS -->
+
+    <!-- New group Project Responsible -->
+    <record id="group_project_responsible" model="res.groups">
+        <field name="name">Responsible</field>
+        <field name="category_id" ref="base.module_category_project_management"/>
+        <field name="implied_ids" eval="[(6, 0, [ref('project.group_project_user')])]"/>
+    </record>
+
+    <!-- New group End User -->
+    <record id="group_project_end_user" model="res.groups">
+        <field name="name">End User</field>
+        <field name="category_id" ref="base.module_category_project_management"/>
+    </record>
+
+    <!-- Rename group Project Manager to Project Administrator -->
+    <record id="project.group_project_manager" model="res.groups">
+        <field name="name">Administrator</field>
+        <field name="implied_ids" eval="[(6, 0, [ref('group_project_responsible')])]"/>
+    </record>
+
+    <!-- Modify implied_ids -->
+    <record id="project.group_project_user" model="res.groups">
+        <field name="implied_ids" eval="[(6, 0, [ref('group_project_end_user')])]"/>
```

Comment:
Won't this make Project Users inherit unwanted record rules?
IMO Project End Users should inherit from Project Users, and then add record rule limitations - AFAIK record rules override base ACLs.